src/solc_0.8/common/interfaces/IPolygonAssetERC721.sol

```diff
@@ -15,4 +15,49 @@ interface IPolygonAssetERC721 is IERC721Base {
     function setTokenURI(uint256 id, string memory uri) external;
 
     function tokenURI(uint256 id) external view returns (string memory);
+
+    function approveFor(
+        address from,
+        address operator,
+        uint256 id
+    ) external override;
+
+    function setApprovalForAllFor(
+        address from,
+        address operator,
+        bool approved
+    ) external override;
+
+    function burnFrom(address from, uint256 id) external override;
+
+    function burn(uint256 id) external override;
+
+    function safeTransferFrom(
+        address from,
+        address to,
+        uint256 tokenId
+    ) external override;
+
+    function batchTransferFrom(
+        address from,
+        address to,
+        uint256[] calldata ids
+    ) external override;
+
+    function safeBatchTransferFrom(
+        address from,
+        address to,
+        uint256[] calldata ids,
+        bytes calldata data
+    ) external override;
+
+    function exists(uint256 tokenId) external view override returns (bool);
+
+    function supportsInterface(bytes4 id) external view override returns (bool);
+
+    function setTrustedForwarder(address trustedForwarder) external override;
+
+    function isTrustedForwarder(address forwarder) external override returns (bool);
+
+    function getTrustedForwarder() external override returns (address trustedForwarder);
```

Comment:
IERC721Base already have all those methods declared
why re-add them here ?

Reply:
We needed to rewrite the IERC721Base code again so that the following error could be resolved.
![needtorewitecodeinparent](https://user-images.githubusercontent.com/91669586/176430511-535057dd-40ba-45cc-8e1f-9c935f407f0e.png)
I tried removing the code but the issue still persists.